docs/modules/c-sharp/quickstart.md

```diff
@@ -250,12 +265,16 @@ From the `quickstart-chat` directory, run:
 spacetime publish --project-path server <module-name>
 ```
 
+```bash
+npm i wasm-opt -g
+```
+
 ## Call Reducers
 
 You can use the CLI (command line interface) to run reducers. The arguments to the reducer are passed in JSON format.
 
 ```bash
-spacetime call <module-name> send_message '["Hello, World!"]'
+spacetime call <module-name> send_message "Hello, World!"
```

Comment:
I assume it caused some cryptic error with the square brackets? :sweat_smile:

Reply:
'twas a cryptic err ;)